R/tests.R

```diff
@@ -259,6 +259,39 @@ testPlots <- function() {
                    xlab = "PC 1", ylab = "PC 2", main = "PCA")
   }
   assign("plotMultiFeature", plotMultiFeature, envir = parent.frame())
+  plotMultiTest_sf <- function(x) {
+    var_x <- data.frame(lapply(x$results, `[`, 2))
+    colnames(var_x)<- names(x$results)
+    features <- rownames(var_x)
+    var_x <- data.frame(features, gather(var_x))
+
+    var_y <- data.frame(lapply(x$results, `[`, 3))
+    colnames(var_y) <- names(x$results)
+    features <- rownames(var_y)
+    var_y <- data.frame(features, gather(var_y))
```

Comment:
The tests are failing because of this call to `gather()`. I assume you are calling `tidyr::gather()`. tidyr isn't a dependency of OmicNavigator, so we can't use it in the test examples. Could you replace this with either `stats::reshape()` or `data.table::melt()`?